contracts/debtAllocators/GenericDebtAllocator.sol

```diff
@@ -108,9 +191,22 @@ contract GenericDebtAllocator is Governance {
         // Make sure we have a target debt.
         require(config.targetRatio != 0, "no targetRatio");
 
+        if (block.timestamp - config.lastUpdate <= minimumWait) {
+            return (false, "min wait");
```

Comment:
do we need to cast the string to `bytes()` like we do above?

Reply:
https://github.com/yearn/vault-periphery/pull/26/commits/19bad7e0777fdeca933c2230cbc2e0dbad15428a